contrib_bots/lib/hangman.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
```

Comment:
······+-----+
······|········|
······0·······|
·····/|\······|
·····/ \······|
···············|
=========
Is part of the message that the bot returns when you lose. At first, when making these "arts", I tried it with spaces instead of these ·'s, but in Zulip, when you type "a        b" it will show up in the chat as "a b". So spaces wouldn't work. I searched for a character that was small and made it so that you could still see what it was trying to message. I found these ·'s and decided to use them. Without the # -*- coding: utf-8 -*-, these characters would crash the bot.